docs/upgrade/17_to_18/mysql5.sql

```diff
@@ -0,0 +1,3 @@
+-- Drop table so it can be recreated.  Previous functionality was unused, and broken even if it was
+drop table oc_user_settings;
```

Comment:
Makes sense to drop this, but could also accidentally cause problems if adopters don't run the migration exactly at the correct time (thinking about adopters upgrading over multiple versions, …).

That could be easily avoided by naming the new table differently. Maybe something like `oc_frontend_user_settings`? What do you think?

Reply:
I'm fine with renaming it, but do we leave this in so that the defunct table *hopefully* goes away?

Reply:
https://github.com/opencast/opencast/pull/6443 introduced DB migration scripts for mariadb and postgres. I think we could add the drop table there to avoid having multiple different scripts right?